configurator/config.json

```diff
@@ -18,11 +18,12 @@
     "certfile": "fullchain.pem",
     "keyfile": "privkey.pem",
     "ssl": false,
-    "allowed_networks": ["192.168.0.0/16"],
+    "allowed_networks": ["192.168.0.0/16", "172.30.0.0/16"],
     "banned_ips": ["8.8.8.8"],
     "banlimit": 0,
     "ignore_pattern": ["__pycache__"],
-    "dirsfirst": false
+    "dirsfirst": false,
+    "enforce_basepath": false
```

Comment:
What does this do?  I don't see it mentioned in the docs?

Reply:
It's not really relevant for the hassio version of the addon because it already is inside a container. It's targeted at standalone installations where in theory the whole filesystem would be accessible.

Reply:
In this case, we should remove it from user configuration

Reply:
Ok. Version 0.3.0 is almost done. I'll remove it with the next upgrade.